boards/longan_nano/board.asm

```diff
@@ -1,4 +1,4 @@
-# include definitions related to the GigaDevice GD32VF103 chip
+# include definitions related to the GigaDevice GD32VF103CBT6 chip
```

Comment:
The file being included here contains definitions common to all chips within the GD32VF103 family. Specific differences between chips (CBT6 vs C8T6) are expressed in these separate board files. Therefore I think it makes more sense to leave this comment as referring to the generic chip family name.

```suggestion
# include definitions related to the GigaDevice GD32VF103 chip
```

Reply:
Oh right, got it, thanks!